salt/modules/yumpkg.py

```diff
@@ -1033,7 +1068,8 @@ def install(name=None,
         __salt__['cmd.run'](cmd, output_loglevel='trace')
 
     if to_reinstall:
-        cmd = 'yum -y {repo} {exclude} {branch} {gpgcheck} reinstall {pkg}'.format(
+        cmd = '{yum_command} {repo} {exclude} {branch} {gpgcheck} reinstall {pkg}'.format(
```

Comment:
This command previously used the `-y` flag to avoid confirmation prompts. Was that removed intentionally here?